Makefile

```diff
@@ -158,13 +158,17 @@ help:
 	@echo 'OPTIONS:'
 	@echo ''
 	@echo '  FLAVORS - ceph-container images to operate on in the form'
-	@echo '    <CEPH_VERSION>,<BASEOS_REPO>,<BASEOS_TAG>'
+	@echo '    <CEPH_VERSION>[CEPH_POINT_RELEASE],<BASEOS_REPO>,<BASEOS_TAG>'
 	@echo '    and multiple forms may be separated by spaces.'
 	@echo '      CEPH_VERSION - named ceph version (e.g., luminous, mimic)'
-	@echo '      BASEOS_REPO - The base image to use for the daemon-base container. This is also'
-	@echo '                    the distro path sourced from ceph-container (e.g., ubuntu, centos)'
-	@echo '      BASEOS_TAG - Tagged version of the base repo to use. Also the distro version'
-	@echo '                   sourced from ceph-container (e.g., ubuntu:"16.04", centos:"7")'
+	@echo '      CEPH_POINT_RELEASE - Optional field to select a particular version of Ceph'
+	@echo '                           Regarding the package manager the version separator may vary :'
```

Comment:
For a user who doesn't know that the `CEPH_POINT_RELEASE` field is directly appended to package names, this could still be confusing. It may be best just to say explicitly that the CPR value (including the separator '-' or '=') will be appended to each ceph package name to set a specific version. And that the package specification separator for yum/dnf/zypper is a dash, and apt is equal. It's verbose, but it provides necessary clarity.

Reply:
This documentation is not about how it is implemented but what is the goal of that field. For the user, this is enough information to understand that CPR is used to pin to a particular release.

Reply:
The user needs enough information to use the field effectively. It would be unclear when to pick `-` vs `=` for a package manager that we don't already have in the codebase unless the user understands that all ceph packages will have the field appended including what we've called the separator.